frontend/components/ModalContainer.jsx

```diff
@@ -71,17 +86,29 @@ ModalContainer.propTypes = {
   })).isRequired,
   isLoading: PropTypes.bool.isRequired,
   onCloseFAQ: PropTypes.func.isRequired,
+  onConfirmAgeConsent: PropTypes.func.isRequired,
+  onRejectAgeConsent: PropTypes.func.isRequired,
   FAQModalVisible: PropTypes.bool.isRequired,
+  AgeConsentModalVisible: PropTypes.bool.isRequired,
```

Comment:
It's a nitpick but we're getting pretty inconsistent with our prop and selector naming conventions, it might be worth spinning through to clean those up later on. What we'd been doing initially was UpperCamelCase for components, and lowerCamelCase for props, with functions labeled `onWorkflowEvent` and booleans `isSomeState`.

The selectors introduced some confusion, because my naming of them was inconsistent. After reviewing some other projects I think the most comprehensible approach—speaking strictly for myself—would be to name all selectors `selectThingYouAreLookingForInTheState`, like `selectAgeConsentModalVisibility`.

I can open an issue in the backlog to scrub through all this later on, it doesn't need to hold this PR up or be bikeshedded now.